03_Modules/SmartCharging/src/module/api.ts

```diff
@@ -547,12 +527,40 @@ export class SmartChargingModuleApi
     CallAction.GetCompositeSchedule,
     GetCompositeScheduleRequestSchema,
   )
-  getCompositeSchedule(
+  async getCompositeSchedule(
     identifier: string,
     tenantId: string,
     request: GetCompositeScheduleRequest,
     callbackUrl?: string,
   ): Promise<IMessageConfirmation> {
+    // OCPP 2.0.1 Part 2 K08.FR.05
+    const evse =
+      await this._module.deviceModelRepository.findEvseByIdAndConnectorId(
+        request.evseId,
```

Comment:
If the incoming request has the EVSEId 0, for getting the composite schedule of the entire station/grid connection as described in `K08.FR.03` this will cause an issue right?

Reply:
Nice catch! Thank you!